gc/verbose/handler_standard/VerboseHandlerOutputStandard.hpp

```diff
@@ -136,8 +143,6 @@ class MM_VerboseHandlerOutputStandard : public MM_VerboseHandlerOutput
 	 */
 	void handleScavengePercolate(J9HookInterface** hook, uintptr_t eventNum, void* eventData);
 	
-	virtual const char *getConcurrentTypeString() { return "scavenge"; }
```

Comment:
`getCycleType` won't work because of this: https://github.com/eclipse/openj9/pull/12247#discussion_r597226927 

Will need to leave this `getConcurrentTypeString` , you should change it it to do something similar to `getCycleType` but only for `OMR_GC_CYCLE_TYPE_SCAVENGE` and `OMR_GC_CYCLE_TYPE_GLOBAL`